pennylane/transforms/condition.py

```diff
@@ -60,11 +61,32 @@ def __init__(
 
 
 def cond(condition, true_fn, false_fn=None):
-    """Condition a quantum operation on the results of mid-circuit qubit measurements.
+    """Quantum-compatible if-else conditionals --- condition quantum operations
+    on parameters such as the results of mid-circuit qubit measurements.
 
-    Support for using :func:`~.cond` is device-dependent. If a device doesn't
-    support mid-circuit measurements natively, then the QNode will apply the
-    :func:`defer_measurements` transform.
+    When used with the :func:`~.qjit` decorator with a hybrid

```

Comment:
I would be first speak about the Python cond and then about the Catalyst one. I don't think the python version should only be a note.